src/App.vue

```diff
@@ -19,6 +20,11 @@
         components: {
             Calendar
         },
+        methods: {
+            eventAdded(eventData) {
+                this.events.push(eventData);
+            },
+        },
         mounted() {
             let me = this;
             setTimeout(function () {
```

Comment:
Instead of the two omissions above please use the following code:

`this.$root.$on(EVENT_ADDED, function(payload) {
                me.events.push(payload);
            });`